conf/base.config

```
@@ -126,7 +126,7 @@ process {
     }
 
     withName: RGI_MAIN {
-        memory = { check_max( 4.GB * task.attempt, 'memory'  ) }
+        memory = { check_max( 8.GB * task.attempt, 'memory'  ) }
```

with 12 GB the test-files passed.
```suggestion
        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
```
```suggestion
        memory = { check_max( 8.GB * task.attempt, 'memory'  ) }
```